script.js

```diff
@@ -49,4 +49,90 @@ const photos = [
     },
 ];
 
-console.log('hello');
\ No newline at end of file
+const section = document.getElementById('gallery');
+const more_photos = document.getElementById('loadMore');
+// filter buttons
+const allButton = document.getElementById('all');
+const cityButton = document.getElementById('city');
+const natureButton = document.getElementById('nature');
+const animalButton = document.getElementById('animals');
+//filer classes
+const all_photos = document.getElementsByClassName('all');
+const city_photos = document.getElementsByClassName('city');
+const nature_photos = document.getElementsByClassName('nature');
+const animal_photos = document.getElementsByClassName('animals');
+// photo counter
+let total_photos_loaded = 0;
+
+function loadFirstPhotos(gallery){
+    for (let i = 0; i < 6; i++) {
+        let image = new Image();
+        image.src = gallery[i].url;
+        section.appendChild(image);
+        image.classList.add(gallery[i].type);
+        image.classList.add('all');
+    }
+    total_photos_loaded = 6;
+    console.log('first 6 photos loaded');
+};
+
+function morePhotos(gallery){
+    for (let j = total_photos_loaded+1 ; j < total_photos_loaded+6; j++) {
+        let image = new Image();
+        image.src = gallery[j].url;
+        section.appendChild(image);
+        image.classList.add(gallery[j].type);
+        image.classList.add('all');
+    }
+    total_photos_loaded = total_photos_loaded+6;
+    console.log('6 more photos loaded');
+};
+
+
+function filterPhotos(keyword){
+    if (keyword == 'city'){
```

Comment:
This is a great opportunity to use a `switch` statement, like we've looked at in class. 

Also, instead of having to create a `for` loop for every array and hiding the element, why not use your `all_photos` array to hide all the photos, and then loop through the array of the photo type that was selected. This would clean up your code a lot.

Reply:
I added the switch statement, and it does look a lot neater. When I first learned JavaScript I wasn't taught about switch statements so it completely slipped my mind.